uncompr.c

```diff
@@ -3,7 +3,6 @@
  * For conditions of distribution and use, see copyright notice in zlib.h
  */
 
-#define Z_INTERNAL
```

Comment:
Could removal of this definition be causing the problem?

Reply:
Z_INTERNAL decorates a bunch of internal functions.  Yes, removing that would cause a whole host of issues, including breaking compilation everywhere it's used.